genesyscloud/webdeployments_configuration/resource_genesyscloud_webdeployments_configuration_test.go

```diff
@@ -1157,3 +1164,27 @@ func verifyConfigurationDestroyed(state *terraform.State) error {
 
 	return nil
 }
+
+func cleanupWebDeploymentsConfiguration() error {
+	config, err := provider.AuthorizeSdk()
+	if err != nil {
+		return err
+	}
+	deploymentsAPI := platformclientv2.NewWebDeploymentsApiWithConfig(config)
+
+	configurations, resp, getErr := deploymentsAPI.GetWebdeploymentsConfigurations(false)
+	if getErr != nil {
+		return util.BuildWithRetriesApiDiagnosticError(resourceName, fmt.Sprintf("failed to get page of configurations: %v", getErr), resp)
+	}
+
+	for _, configuration := range *configurations.Entities {
+		if configuration.Name != nil && strings.HasPrefix(*configuration.Name, "Test Configuration ") {
+			resp, delErr := deploymentsAPI.DeleteWebdeploymentsConfiguration(*configuration.Id)
+			if delErr != nil {
+				return util.BuildWithRetriesApiDiagnosticError(resourceName, fmt.Sprintf("Failed to delete configuration %s: %s", *configuration.Id, delErr), resp)
+			}
```

Comment:
I would also log here instead of return an error. There could be something weird going on with one configuration in particular - doesn't mean we shouldn't try to cleanup the other ones.